packages/components/multiselect/src/Multiselect.tsx

```diff
@@ -133,18 +175,19 @@ function _Multiselect(props: MultiselectProps, ref: React.Ref<HTMLDivElement>) {
     [onSearchValueChange, setSearchValue],
   );
 
-  const resetSearch = () => {
+  const resetSearchInput = useCallback(() => {
+    if (!searchValue) return;
```

Comment:
as the button is inactive when there is no search value, this is a little double trouble?

Reply:
This logic already existed before but below when calling`resetSearch`. I just wanted to put this in a `useCallback` and clean up the render method. We can also drop it if we don't need it anymore. Only case to have this is that we also enforce programmatically this behaviour while you can just remove the `disabled` tag via the devtools in the browser.